### DISCUSSION TITLE
Follow-Up Point de situation GitHub

### DISCUSSION THREAD

**Discussion by christianheimann** [e-id-admin/governance-sounding-board#30] [General]
Gedanken, Anmerkungen, Kritik, Anregungen zur Präsentation "Point de situation GitHub"

**Comment by kgerd** [comment]
Es wurde um Use Cases gebeten. Wir haben ein paar für die Hochschulbildung zusammengestellt: https://openlcms.org/use-cases/

Persönlich denke ich, dass das Projekt realistischer wird, wenn man davon ausgeht, dass viele, weitgehend unabhängige SSI-Ökosysteme existieren - die obigen Use Cases wären in einem Ökosystem der Schweizer Hochschulen.

Die E-ID müsste dabei nur relativ wenig leisten: sie käme als Verifiable Credential  (VC) daher, mit dem sich jemand bestätigen lassen kann, dass ein oder mehrere Decentralized Identifiers (DIDs) tatsächlich seiner oder ihrer Identity zugeordnet sind (Identity Assertion). In anderen Worten: alles, was ich mir von einer E-ID versprechen würde, ist ein staatlich bestätigtes VC, das ich in irgendein Wallet stecken kann. Das staatliche Ökosystem wäre überschaubar: es hat ein paar Behörden, die VCs ausstellen können, und ein Verifiable Data Registry ("Ledger"), wo man die VCs verifizieren kann.

Wenn ich in irgendeinem Ökosystem tatsächlich beweisen muss, dass ich (vertreten durch eine DID) auch tatsächlich staatlich verifizierbar "ich" bin, reiche ich das VC ein (statt meines Passes oder Personalausweises). Jedes Ökosystem kann für sich entscheiden, welche VCs aus welchen Ökosystemen es zur Identity Assertion akzeptiert (das ist Teil der Governance  der jeweiligen System) - die meisten Ökosystem würden vermutlich auch VCs von anderen Staaten annehmen, oder - in unserem Fall - VCs von Hochschulkanzleien.
